web/src/locales/zh/data.json

```diff
@@ -167,13 +167,15 @@
     "Expire time - Tooltip": "到期时间（留空表示永久有效）",
```

Comment:
About how to update i18n, see: https://casdoor.org/docs/contributing/#how-to-update-i18n-data